.github/workflows/docker-build-push.yaml

```
@@ -91,7 +91,7 @@ jobs:
       - name: Download Single-Platform Packages
         uses: actions/download-artifact@v4
         with:
-          path: .
+          path: "*.xpkg"
           merge-multiple: true
```

Please try the following:

```suggestion
          path: "."
          merge-multiple: true
          pattern: "!*.dockerbuild" # This gets uploaded by docker/build-push-action but must be skipped: https://github.com/actions/toolkit/pull/1874
```